Cargo.toml

```diff
@@ -159,6 +159,8 @@ members = [
     "state-sync/inter-component/mempool-notifications",
     "state-sync/state-sync-v1",
     "state-sync/state-sync-v2",
+    "state-sync/storage-service/server",
+    "state-sync/storage-service/types",
```

Comment:
Yeah, it's not great. But, I'd argue that this is a limitation with the logger and setup and not with our code structure. For example, it's important to split the code here up because of circular dependencies and not wanting to tie the server and client together (which is what state sync currently does). When I come to add the logger I'll try to dig a little